### DISCUSSION TITLE
Improve Annotations

### DISCUSSION THREAD

**Discussion by Lazaro10e0** [Cimbali/pympress#288] [General]
I love this tool since it allows me to present perfectly my latex beamer slides. The only problem is the quality of my annotations. I use a tablet to draw in the slides while presenting, but I find it quite difficult to write with good quality. Not only because it doesn't detect thickness/pressure, also the minimum thickness is very thick and the path is not exactly what is being drawn. 

If we cauld improve this it good be more than perfect tool!!
Thanks

**Comment by Cimbali** [comment]
I think changing the minimum thickness is quite easy to do. What size do you want to draw/write? Can you share a screenshot maybe? This could also help with the path not being exactly what is drawn.

As for thickness/pressure I can look into what’s supported by Gtk but not sure there is much we can do.

**Reply by Lazaro10e0** [reply]
Changing the minimum thickness would be a nice improvement, mainly when writting small stuff. 
I attach an image where you can see in the spiral it is not smooth, it looks like it is done with small straight lines. I can notice perfectly that this happens when you write it quickly. If you do it slowly it seems very smooth.

![image](https://github.com/Cimbali/pympress/assets/38687716/a94dc6f7-ff52-4fbd-99de-faa34ede0f58)
